sql-plugin/src/main/scala/org/apache/spark/sql/rapids/bitwise.scala

```diff
@@ -47,11 +47,11 @@ object ShiftHelper extends Arm {
     }
   }
 
-  def fixupDistanceNoClose(t: DType, distance: Scalar): Scalar = {
+  def fixupDistanceNoClose(t: DType, distance: GpuScalar): Scalar = {
     if (distance.isValid) {
-      Scalar.fromInt(distance.getInt & maskForDistance(t))
+      Scalar.fromInt(distance.getValue.asInstanceOf[Int] & maskForDistance(t))
     } else {
-      distance.incRefCount()
+      distance.getBase.incRefCount()
```

Comment:
Seems this is not the case, since it requires a cudf Scalar to be returned. But there are two cases in `nullExpression.scala` and `mathExpressions.scala` where the old `copy ` are called.